projects/components/src/filtering/filter-button-wrapper/filter-button-wrapper.component.ts

```diff
@@ -21,7 +21,6 @@ export class FilterButtonWrapperComponent {
 }
 
 export interface FilterButtonWrapperData {
-  metadata?: FilterAttribute[];
-  attribute?: FilterAttribute;
-  value?: FilterValue;
+  targetAttribute: FilterAttributeExpression;
```

Comment:
@aaron-steinfeld  @palbizu   `targetAttribute` can be null, and we need to handle it. UI code failed at run time, crashing the app and causing prod issues. 

https://traceableai.atlassian.net/browse/ENG-35622
https://traceableai.atlassian.net/browse/ENG-35603

Reply:
Commented on the fix and can continue discussion there https://github.com/hypertrace/hypertrace-ui/pull/2421#discussion_r1334677536